libraries/standard/http/include/http_client.h

```diff
@@ -164,14 +168,47 @@ typedef enum HTTPStatus
      */
     HTTP_NETWORK_ERROR,
 
-    HTTP_NOT_SUPPORTED,
+    /**
+     * @brief Part of the HTTP response was received from the network.
+     *
+     * This can occur only if #HTTPTransportRecv_t returns zero before the full
+     * response message was received.
+     *
+     * Functions that may return this value:
+     * - #HTTPClient_Send
+     */
     HTTP_PARTIAL_RESPONSE,
+
+    /**
+     * @brief No HTTP response was received from the network.
+     *
+     * This can occur only if #HTTPTransportRecv_t returns zero when first
```

Comment:
This seems to be leaking some implementation detail. It could just mention generally that when no data is received from the transport interface.

Reply:
Makes sense I will change it